ios/TabViewImpl.swift

```diff
@@ -90,7 +94,17 @@ private func configureAppearance(for appearanceType: String) -> UITabBarAppearan
   default:
     appearance.configureWithDefaultBackground()
   }
-  
+
+  // @see https://stackoverflow.com/a/71934882
+  if let inactiveTintColor = props.inactiveTintColor {
+    appearance.stackedLayoutAppearance.normal.titleTextAttributes = [.foregroundColor: UIColor(inactiveTintColor)]
```

Comment:
You are applying this to `stackedLayoutAppearance` I think we need to apply it to every appearance. This might stop working when phone is rotated.

Reference: https://emptytheory.com/2019/12/31/using-uitabbarappearance-for-tab-bar-changes-in-ios-13/